docs/docs/basic-demos/fraud-detection.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 3
+---
+
+# Fraud Detection
+
+Demo 03
```

Comment:
I assume this has to be filled in manually.

Reply:
Yes